src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig

```diff
@@ -196,14 +196,14 @@
     </div>
     <div class="mt-2 info-block">
       <div class="row">
+        {% set isPrivateNoteOpen = not orderForViewing.customer.privateNote is empty %}
+
         <div class="col-md-6">
-          <h3 class="mb-0">
+          <h3 class="mb-0 {{ not isPrivateNoteOpen ? 'd-print-none': '' }}">
```

Comment:
```suggestion
          <h3 class="mb-0{{ not isPrivateNoteOpen ? ' d-print-none': '' }}">
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479